src/main/java/uk/gov/hmcts/ccd/domain/model/definition/WizardPageComplexFieldOverride.java

```diff
@@ -60,19 +61,19 @@ public void setShowCondition(String showCondition) {
         this.showCondition = showCondition;
     }
 
-    public DisplayContext displayContextType() {
-        return Optional.ofNullable(getDisplayContext())
-            .filter(dc -> !dc.equals("HIDDEN"))
-            .map(DisplayContext::valueOf)
-            .orElse(null);
-    }
-
-    @JsonProperty("default_value")
```

Comment:
Changed to JsonIgnore instead of removing, because it is defined in the definition-store and comes from there.